jobs/ci-run/integration/common/test-runner.sh

```diff
@@ -69,6 +69,20 @@ while [ $attempts -lt 3 ]; do
         echo -e "[default]\nregion = us-east-1" > "$HOME"/.aws/config
         chmod 600 ~/.aws/*
     fi
+    # shellcheck disable=SC2193
+    if [ "${{BOOTSTRAP_PROVIDER:-}}" = "azure" ]; then
+        if [ ! "$(which az >/dev/null 2>&1)" ]; then
+            curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+        fi
+
+        app_id=$(cat "$HOME"/.local/share/juju/credentials.yaml | grep azure: -A 4 | grep application-id: | awk '{print $2}')
+        app_pass=$(cat "$HOME"/.local/share/juju/credentials.yaml | grep azure: -A 4 | grep application-password: | tail -1 | awk '{print $2}')
+        # (anvial) as we can get tenant-id only after we authorize via the browser and check the output of the command:
+        # `az account show`, we need to hardcode it. Taking in consideration that it's constant for our juju-qa azure user.
+        # TODO: (anvial) find a way to get tenant-id automatically
+        tenant_id="40a524d9-f848-46d4-a96f-be6df491fe15"
```

Comment:
Can we put this somewhere like cloud-city, just so it's not available in clear text in a public repository?